tests/operators/test_read_shapefile.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 Met Office and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Reading operator tests."""
+
+
+def test_read_shapefile():
+    """Verifies the shapefile is read in correctly."""
+    return None
```

Comment:
```suggestion
import pytest


@pytest.mark.skip(reason="Shapefile support not yet implemented.")
def test_read_shapefile():
    """Verifies the shapefile is read in correctly."""
    raise NotImplementedError
```
This will [skip the test](https://docs.pytest.org/en/stable/skipping.html), which means it won't misleadingly show up as passing. Raising the NotImplementedError means that if the test *does* get run it will fail.

Reply:
I might just delete the test and the `shapefiles/` in the test dir since it's no longer being used.